src/Glue.elm

```diff
@@ -130,29 +130,43 @@ poly :
     , set : subModel -> model -> model
     , init : ( subModel, Cmd msg )
     , update : subMsg -> subModel -> ( subModel, Cmd msg )
-    , view : subModel -> Html msg
     , subscriptions : subModel -> Sub msg
     }
     -> Glue model subModel msg subMsg
-poly { get, set, init, update, view, subscriptions } =
+poly { get, set, init, update, subscriptions } =
     Glue
         { model = set
         , init = init
         , update =
             \subMsg model ->
                 get model
                     |> update subMsg
-        , view =
-            \model ->
-                get model
-                    |> view
+        , view = \model -> text ""
```

Comment:
I think we should remove this from type itself. Mocking something implicitly seems to be confusing to me.